src/test/java/seedu/storage/StorageTest.java

```diff
@@ -0,0 +1,40 @@
+package seedu.storage;
+
+import org.junit.jupiter.api.Test;
+import seedu.data.exception.FridgetException;
+import seedu.notification.Notification;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class StorageTest {
+    private static final String FILE_PATH_LIST = "config/savedList.txt";
+    private static final String FILE_PATH_LOGS = "config/savedLogs.txt";
```

Comment:
Not sure if you want to put in another directory but this should be fine too. Maybe can rename as testSavedList/Log